content/pt-br/blog/_posts/2022-02-17-updated-dockershim-faq.md

```diff
@@ -0,0 +1,208 @@
+ferramentas a crescer e prosperar no nosso ecossistema, alguns substituindo
+aspectos do Docker, enquanto outros aprimoram funcionalidades já existentes.
+
+### Existem exemplos de pessoas que usam outros agentes de execução de contêineres em produção hoje?
+
+Todos os artefatos produzidos pelo projeto Kubernetes (binários Kubernetes) são validados
+a cada lançamento de versão.
+
+Além disso, o projeto [kind] vem usando containerd há algum tempo e tem
+visto uma melhoria na estabilidade para seu caso de uso. Kind e containerd são executados
+várias vezes todos os dias para validar quaisquer alterações na base de código do Kubernetes.
+Outros projetos relacionados seguem um padrão semelhante, demonstrando a estabilidade e
+usabilidade de outros agentes de execução de contêiner. Como exemplo, o OpenShift 4.x utiliza
+o agente de execução [CRI-O] em produção desde junho de 2019.
+
+Para outros exemplos e referências, dê uma olhada em projetos adeptos do containerd e
+CRI-O, dois agentes de execução de contêineres sob o controle da _Cloud Native Computing Foundation_
+([CNCF]).
+
+- [containerd](https://github.com/containerd/containerd/blob/master/ADOPTERS.md)
+- [CRI-O](https://github.com/cri-o/cri-o/blob/master/ADOPTERS.md)
+
+[CRI-O]: https://cri-o.io/
+[kind]: https://kind.sigs.k8s.io/
+[CNCF]: https://cncf.io
+
+### As pessoas continuam referenciando OCI, o que é isso?
+
+OCI significa _[Open Container Initiative]_ (ou Iniciativa Open Source de Contêineres), que padronizou muitas das
+interfaces entre ferramentas e tecnologias de contêiner. Eles mantêm uma
+especificação padrão para imagens de contêiner (OCI image-spec) e para 
+contêineres em execução (OCI runtime-spec). Eles também mantêm uma implementação real
+da especificação do agente de execução na forma de [runc], que é o agente de execução padrão 
+para ambos [containerd] e [CRI-O]. O CRI baseia-se nessas especificações de baixo nível para
+fornecer um padrão de ponta a ponta para gerenciar contêineres.
+
+[Open Container Initiative]: https://opencontainers.org/about/overview/
+[runc]: https://github.com/opencontainers/runc
+[containerd]: https://containerd.io/
+
+### Qual implementação de CRI devo usar?
+
+Essa é uma pergunta complexa e depende de muitos fatores. Se você estiver
+trabalhando com Docker, mudar para containerd deve ser uma troca relativamente fácil e
+terá um desempenho estritamente melhor e menos sobrecarga. No entanto, nós encorajamos você a
+explorar todas as opções do [cenário CNCF], pois outro agente de execução de contêiner
+pode funcionar ainda melhor para o seu ambiente.
+
+[cenário CNCF]: https://landscape.cncf.io/card-mode?category=container-runtime&grouping=category
+
+### O que devo ficar atento ao mudar a minha implementação de CRI utilizada?
+
+Embora o código de conteinerização base seja o mesmo entre o Docker e a maioria
+CRIs (incluindo containerd), existem algumas poucas diferenças. Alguns
```

Comment:
```suggestion
dos CRIs (incluindo containerd), existem algumas poucas diferenças. Alguns
```

Reply:
Thanks @edsoncelio! I've just submitted a new commit including this correction :smile:

Reply:
Thanks @edsoncelio! I've just submitted a new commit including this correction :smile: